crates/database/src/migrations/sql/V12__add_model_settings.sql

```diff
@@ -0,0 +1,17 @@
+-- Create model_settings table
+-- This table stores admin-level model settings per model as JSONB to allow flexible schema evolution
+CREATE TABLE model_settings (
```

Comment:
Can we use `models` here? The table should be the only one that stores the data and configs about models.